docs/filters.md

```diff
@@ -108,6 +108,8 @@ This will not work for ambiguous expressions (containing OR or NOT operators) an
 - [`is`](#is)
 - [`created`](#created)
 - [`updated`](#updated)
+- [`updatedByOthers`](#updatedByOthers)
+- [`updatedBySelf`](#updatedBySelf)
```

Comment:
Didn't we decide to use `-word` format instead, which is similar to git command format?